ontouml-catalog/.htaccess

```diff
@@ -0,0 +1,14 @@
+RewriteEngine on
+
+# Redirection Match rule for accessing the catalog repository git page
+
+RedirectMatch 302 ^/?$ https://github.com/unibz-core/ontouml-models
```

Comment:
mod_alias RedirectMatch checks against the full path, so you'd need to add /ontouml-catalog at the front of all these.  You could also change to use mod_rewrite's RewriteRule which matches on the relative part.